tests/specs/graph/JsrSpecifiers_FastCheck_Vars.txt

```diff
@@ -35,6 +35,11 @@ export const asIs2 = {
   g: 1++,
   h: [...[1, 2], ...["test"]],
 };
+export const func1 = function (): string { return "test" };
+export const func2 = (): string => "test";
+export const func3 = () => "test";
+export const func4 = () => "test" as const;
```

Comment:
We should add a test like the following in order to ensure the dep analysis is working for functions and arrows in variables too:

```ts
export const func5 = (arg: Public1) => "test" as const;
export const func6 = function(): Public2 { return new Public2(); };

interface Public1 {}
class Public2 {}
```